src/theme/theme.ts

```diff
@@ -1347,12 +1347,12 @@ const fixedTokens = {
     active: '&:active',
     placeholder: '&::placeholder',
     disabled: '&:disabled, &[aria-disabled=true], &[data-state="disabled"]',
-    selected: '&[data-state="selected"]',
+    selected: '&[data-state="selected"], &[aria-selected=true]',
```

Comment:
Why is that necessary here? We should store the selected state in the `data-selected` attribute. @adrian-potepa wdyt